.gitignore

```diff
@@ -1,4 +1,6 @@
 node_modules
+!lib/__tests__/fixtures/node_modules
```

Comment:
medium

remove this line

Reply:
There is https://github.com/stylelint/stylelint/blob/main/lib/__tests__/fixtures/node_modules/test.css actually.

Reply:
So you are saying it was an oversight of be558d4.

https://github.com/stylelint/stylelint/blob/d2c8d0dd20924706c3075c9d208234c373a41f42/lib/__tests__/ignore.test.mjs#L386